__tests__/endpoints.test.js

```diff
@@ -180,4 +180,121 @@ describe("/api/reviews/:review_id/comments", () => {
         expect(response.body.message).toBe("Review not found :(");
       });
   });
+  test("GET - status 400, responds with error message when passed review number is invalid", () => {
+    return request(app)
+      .get("/api/reviews/something/comments")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.message).toBe("Bad request :(");
+      });
+  });
+});
+describe("POST, /api/reviews/:review_id/comments", () => {
+  test("POST - status 201 - responds with fresh-posted comment", () => {
+    const newComment = {
+      username: "dav3rid",
+      body: "OMG this game is so good I spent 2 days with no rest playing it",
+    };
+    return request(app)
+      .post("/api/reviews/5/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        const comment = response.body.comment;
+        expect(comment.review_id).toBe(5);
+        expect(comment.author).toBe("dav3rid");
+        expect(comment.body).toBe(
+          "OMG this game is so good I spent 2 days with no rest playing it"
+        );
+        expect(comment.votes).toBe(0);
+      });
+  });
+  test("POST - status 201 - responds with fresh-posted comment if additional property passed except of necessary", () => {
+    const newComment = {
+      username: "dav3rid",
+      body: "OMG this game is so good I spent 2 days with no rest playing it",
+      votes: 10,
+    };
+    return request(app)
+      .post("/api/reviews/5/comments")
+      .send(newComment)
+      .expect(201)
+      .then((response) => {
+        const comment = response.body.comment;
+        expect(comment.review_id).toBe(5);
+        expect(comment.author).toBe("dav3rid");
+        expect(comment.body).toBe(
+          "OMG this game is so good I spent 2 days with no rest playing it"
+        );
+        expect(comment.votes).toBe(0);
+      });
+  });
+  test("POST - status 400 - responds with error message when one or more fields of comment object missing", () => {
+    const newComment = {
+      username: "dav3rid",
+    };
+    return request(app)
+      .post("/api/reviews/5/comments")
+      .send(newComment)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.message).toBe(
+          "Now enough information to post a comment"
+        );
+      });
+  });
+  test("POST - status 400 - responds with error message when one or more fields of comment object have incorrect type", () => {
+    const newComment = {
+      username: "dav3rid",
+      body: 8,
+    };
+    return request(app)
+      .post("/api/reviews/5/comments")
+      .send(newComment)
+      .expect(400)
+      .then((response) => {
+        expect(response.body.message).toBe(
+          "The type of information you are trying to enter are not correct"
+        );
+      });
+  });
+  test("POST - status 400 - responds with error message when non-existent username passed", () => {
```

Comment:
This should be expecting a 404 here for a username that doesn't exist, please change this 👍